frontend/packages/gitops-plugin/src/components/list/GitOpsTableRow.tsx

```diff
@@ -3,32 +3,110 @@ import i18n from '@console/internal/i18n';
 import * as classNames from 'classnames';
 import { GitOpsAppGroupData } from '../utils/gitops-types';
 import { RowFunction, TableData, TableRow } from '@console/internal/components/factory';
-import { Button } from '@patternfly/react-core';
-import { history, ExternalLink } from '@console/internal/components/utils';
+import { Button, Flex, FlexItem, Split, SplitItem, Tooltip } from '@patternfly/react-core';
+import { history, ExternalLink, Timestamp } from '@console/internal/components/utils';
 import { routeDecoratorIcon } from '@console/dev-console/src/components/import/render-utils';
+import {
+  GreenCheckCircleIcon,
+  YellowExclamationTriangleIcon,
+  GrayUnknownIcon,
+} from '@console/shared';
+import './GitOpsTableRow.scss';
 
 const tableColumnClasses = [
   classNames('pf-m-width-20'), // Application name
-  classNames('pf-m-width-40'), // Git repository
+  classNames('pf-m-width-30'), // Git repository
   classNames('pf-m-hidden', 'pf-m-visible-on-md', 'pf-m-width-20'), // Environments
-  classNames('pf-m-hidden', 'pf-m-visible-on-lg', 'pf-m-width-20'), // Last deployment
+  classNames('pf-m-hidden', 'pf-m-visible-on-lg', 'pf-m-width-30'), // Last deployment
 ];
 
-const handleClick = (appGroup: GitOpsAppGroupData) => {
```

Comment:
adding `: GitOpsAppGroupData` was a change that @andrewballantyne had suggested on my PR yesterday, what's the reason for removing it?

Reply:
@reginapizza , it was not my intention to change this. Perhaps, when I applied my patch on top of your latest changes did this...I'll add it back.

Reply:
The handleClick method is no longer need because of the above suggestion.